rpcs3/rpcs3qt/game_compatibility.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "stdafx.h"
```

Comment:
```suggestion
#include "util/types.hpp"
```

Reply:
ok, lets see if with `util/type.hpp` it works also on linux

Reply:
it seems the compile error on linux was fixed by the suggested casting:

`item->setData(Roles::DataSizeRole, static_cast<qulonglong>(info.data_size));`